libs/novui/src/recipes/modal.recipe.ts

```diff
@@ -0,0 +1,50 @@
+import { defineSlotRecipe } from '@pandacss/dev';
+import { ModalBaseStylesNames } from '@mantine/core';
+
+const SLOTS: ModalBaseStylesNames[] = ['body', 'header', 'title', 'overlay', 'root', 'content', 'close', 'inner'];
+
+export const MODAL_RECIPE = defineSlotRecipe({
+  className: 'modal',
+  jsx: ['Modal'],
+  slots: SLOTS,
+  base: {
+    root: {
+      colorPalette: 'mode.cloud',
+      borderRadius: '75',
+      position: 'relative',
+      padding: '50',
+      color: 'typography.text.main',
+      width: 'modal.width !important',
+    },
+    inner: {
+      width: '100%',
+    },
+    header: {
+      background: 'modal.background !important',
+      padding: '100 !important',
+    },
+    title: {
+      fontSize: '100',
+      lineHeight: '125',
+      marginBottom: '25',
+    },
+    content: {
+      background: 'modal.background !important',
+      boxShadow: 'medium !important',
+    },
+    overlay: {
+      background: 'surface.page !important',
+      opacity: '40 !important',
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please extract to a `semanticOpacity` token for `overlay`‏